web/pdf_find_controller.js

```diff
@@ -251,6 +251,10 @@ class PDFFindController {
           // Store the pageContent as a string.
           this.pageContents[i] = strBuf.join('');
           extractTextCapability.resolve(i);
+        }, (reason) => {
```

Comment:
In most places in the patch you're using `}).catch((reason) => {`, but not here[1].
So just to satisfy my own curiosity: Is there a particular reason for using this format here?

---
[1] I'm aware that there's a difference in behaviour between the two formats, since `}, (reason) => {` will only catch errors from the original promise whereas `}).catch((reason) => {` will *also* catch errors from the `then` above